indico/cli/i18n.py

```diff
@@ -113,62 +244,241 @@ def _wrapper(**kwargs):
+            javascript = False
+
+    if not plugin_dirs:
+        _indico_command(babel_cmd, python, javascript, react, locale, no_check)
+    else:
+        for plugin_dir in plugin_dirs:
+            _plugin_command(babel_cmd, python, javascript, react, locale, no_check, plugin_dir)
+
+
+def _make_command(group, cmd_name, babel_cmd, func, **kwargs):
+    cmd_group = group.command(cmd_name, short_help=f'Perform {babel_cmd} operation on {cmd_name}')
+
+    @_common_translation_options(**kwargs)
+    def wrapper(**kwargs):
+        func(babel_cmd=babel_cmd, **kwargs)
+
+    return cmd_group(wrapper)
+
+
+_make_command(compile_catalog, 'indico', 'compile_catalog', _command, require_js=False)
+_make_command(extract_messages, 'indico', 'extract_messages', _command)
+_make_command(update_catalog, 'indico', 'update_catalog', _command)
+_make_command(init_catalog, 'indico', 'init_catalog', _command, require_locale=True)
+
+_make_command(compile_catalog, 'plugins', 'compile_catalog', _command, require_js=False,
+              plugins=True)
+_make_command(extract_messages, 'plugins', 'extract_messages', _command, plugins=True)
+_make_command(update_catalog, 'plugins', 'update_catalog', _command, plugins=True)
+_make_command(init_catalog, 'plugins', 'init_catalog', _command, require_locale=True,
+              plugins=True)
+
+_make_command(compile_catalog, 'all-plugins', 'compile_catalog', _command, require_js=False,
+              all_plugins=True)
+_make_command(extract_messages, 'all-plugins', 'extract_messages', _command, all_plugins=True)
+_make_command(update_catalog, 'all-plugins', 'update_catalog', _command, all_plugins=True)
+_make_command(init_catalog, 'all-plugins', 'init_catalog', _command, require_locale=True,
+              all_plugins=True)
+
+
+@cli.command(short_help='Push .pot files to Transifex.')
+def push():
+    """Push .pot files to Transifex."""
+    try:
+        subprocess.run(['tx', 'push', '-s'], check=True)
+    except subprocess.CalledProcessError:
+        click.secho('Error pushing to transifex ', fg='red', bold=True, err=True)
+
+
+@cli.command(short_help='Pull translated .po files from Transifex.')
+@click.argument('languages', nargs=-1, required=False)
+def pull(languages):
+    """Pull translations from Transifex.
+
+    Pulls only the language codes specified in `languages` if present.
+    If no `languages`, it pulls all the languages currently available in the translations folder.
+    To pull a new language available on Transifex, simply pass the new language code in the argument.
     """
-    root_path = 'indico/translations'
+    if not languages:
+        languages = [d for d in os.listdir(TRANSLATIONS_DIR) if os.path.isdir(os.path.join(TRANSLATIONS_DIR, d))]
+    try:
+        for language in languages:
+            subprocess.run(['tx', 'pull', '-l', language, '-f'], check=True)
+            click.secho(f'Translations updated for {language}.', fg='green', bold=True)
```

Comment:
You can pass the languages as a comma separated string to download all of them at once (e.g. `tx pull -l fr_FR,de_DE`). This'll be faster because it pulls the languages in parallel and the output is IMO clearer as well.

```suggestion
        subprocess.run(['tx', 'pull', '-l', ','.join(languages), '-f'], check=True)
        click.secho('Translations updated.', fg='green', bold=True)
```

Reply:
Done